.github/workflows/software-environments-stable.yml

```
@@ -33,14 +33,33 @@ jobs:
       - name: Install Coiled
         run: python -m pip install coiled
 
-      - name: Create Runtime ${{ matrix.runtime-version }} Python ${{ matrix.python-version }} ${{ matrix.server }} software environment
-        env:
-          DASK_COILED__SERVER: ${{ matrix.server }}
-          DASK_COILED__TOKEN: ${{ secrets[matrix.token-name] }}
+      - name: Create Runtime software environment environment variables
```

Similar to comment in the other workflow

```suggestion
      - name: Create environment variables
```